tests/unit/test_steps.py

```diff
@@ -372,6 +367,40 @@ def test_chaining_steps():
     assert s1.next_step == s2
     assert s2.next_step == s3
 
+
+def test_chaining_choice():
+    s1_pass = Pass('Step - One')
+    s2_choice = Choice('Step - Two')
+    s3_pass = Pass('Step - Three')
+
+    chain1 = Chain([s1_pass, s2_choice, s3_pass])
+    assert chain1.steps == [s1_pass, s2_choice, s3_pass]
+    assert s1_pass.next_step == s2_choice
+    assert s2_choice.default == s3_pass
+    assert s2_choice.next_step is None  # Choice steps do not have next_step
+    assert s3_pass.next_step is None
```

Comment:
When writing unit tests for these modules we should consider making assertions on the generated JSON too. One of the main responsibilities is generating ASL so we should make sure it's correct.

Reply:
Agreed! Will include assertions in the generated JSON in the next commit.

Reply:
As discussed offline, there is already a unit test ([test_choice_state_creation](https://github.com/aws/aws-step-functions-data-science-sdk-python/blob/48efb0440fe2cd89757c42b4fa65ea9d851140cf/tests/unit/test_steps.py#L150-L180)) that ensures the Choice state creation generates the expected ASL